### DISCUSSION TITLE
the speech of single letter

### DISCUSSION THREAD

**Discussion by ningning54321** [coqui-ai/TTS#1625] [General]
Is there any way to create speech of single letter?
In my case, I want to spell a work, if I split a word with blank spaces, some garbled voice is produced.

**Comment by neurlang** [comment]
Are you sure the language the model is trained on (English, ...) contains all of the individual letters as single letter words? If not, you will not be able to pronounce them.

**Reply by ningning54321** [reply]
Thanks! I was using its default vocoder in the release models list, it looks like individual letters were not trianed.
